docs/1-concepts.ipynb

```diff
@@ -0,0 +1,349 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Core Concepts\n",
+    "\n",
+    "The purpose of this guide is to explain the core concepts of *scarlet*, how they are used and how they can be extended and customized for more specialized science cases.\n",
+    "\n",
+    "Other resources are:\n",
+    "\n",
+    "1. Our [Quickstart Guide](0-quickstart.rst) shows a typical *scarlet* session.\n",
+    "2. The [API Documentation](api/scarlet.rst) describes modules and classes of the python library.\n",
+    "3. A more in-depth explaination of the mathematics and algorithms used by *scarlet* is in [Melchior et al. 2018](https://arxiv.org/abs/1802.10157)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "\n",
+    "The goal of *scarlet* is to create a model of individual astrophysical sources from a collection of observations of a rectangular region of the sky. These observations can be in multiple filter bands (which we will call \"channels\" internally) with different PSFs, from telescopes with different resolutions, and eventually even spectroscopic instruments.\n",
+    "A main emphasis of *scarlet* lies in deblending overlapping sources. \n",
+    "As is pointed out by Robert Lupton, perfect reconstruction of a blended scene is [impossible](https://docushare.lsst.org/docushare/dsweb/Services/Document-29071), however by making a few minor assumptions *scarlet* improves on other blending algorithms by leveraging as much data as possible.\n",
+    "\n",
+    "The basic assumption of *scarlet* is that the sources in an astrophysical image can be thought of as a collection of `Component` objects, where each component has its own internal representation of the source properties (non-parameteric or parametric), from which it computes its model of the sources in a 3D hyper-spectral data cube of pixels vs wavelegth.\n",
+    "A common model is `FactorizedComponent`, which describes the source as a non-parametric morphology image and an intensity in every channel (SED).\n",
+    "With this ansatz, more complicated objects, like galaxies, can be thought of as a combination multiple components (a `ComponentTree`), where components with different SEDs represent different populations of stars or gases in the host galaxy. To properly separate sources, further assumptions are required, for example that all of the flux is positive and that the spatial distribution stars and galaxies monotonically decrease from their centers.\n",
+    "\n",
+    "A `Frame` contains metadata for the hyperspectral cube *scarlet* seeks to construct as well as those describing an `Observation`. The latter is the combination of a `Frame` with several data units. Each observation can have multiple filter bands with a different PSF in each band that is internally matched to the model PSF.\n",
```

Comment:
I've looked into that, but it seems the only automated way of doing it is translating the notebooks to rst, so that things like `~scarlet.Box` are resolved. This runs counter the focus on notebooks, in particular for growing the tutorials. I have punted in this point, as I didn't want to be bugged down by adding manual links.

Reply:
We had a way to do this before. I think that as long as you specify the full path to the API document it should work. This should at least be a new issue because links to the API in docs is pretty much expected these days.

Reply:
I'm fine with a new PR specifically for the docs/api integration

Reply:
opened issue